### ISSUE TITLE
Erroneous removal of parentheses 

### ISSUE THREAD

**Issue by alimoeeny** [nicklockwood/SwiftFormat#363]
This code:

``` swift
  func tryLoadGameplay(withID gameplayID: String) {
        if let gameplay = (FLGameplayManager.sharedInstance.currentQueue.gamePlays.filter { $0.flid == gameplayID }).first {
            tryLoadGameplay(gameplayID)
        }
    }
```

will gets it's parentheses removed which turns it into:

``` swift
    func tryLoadGameplay(withID gameplayID: String) {
        if let gameplay = FLGameplayManager.sharedInstance.currentQueue.gamePlays.filter { $0.flid == gameplayID }.first {
            tryLoadGameplay(gameplay)
        }
    }
```

Which is no longer a valid code and fails to compile with:
`Anonymous closure argument not contained in a closure` 

Using swift format as a build stage like: 
``` shell
"${PODS_ROOT}/SwiftFormat/CommandLineTool/swiftformat" "${SRCROOT}/../../../Assets/Plugins/."
```
and my version is `SwiftFormat/CLI (0.39.1)`

**Comment by nicklockwood** [comment]
@alimoeeny thanks for reporting. But I think there is a typo in your code sample, as the parentheses aren't balanced. Can you double-check?

**Comment by alimoeeny** [comment]
Thanks @nicklockwood  for your swift reply :) 
This is a typo in me filing the issue and not in the original code, Iet me edit the issue above, and for the record this is from the diff that fixes the problem in my private repo:
``` 
--- a/Assets/Plugins/iOS/Floreo/Tabs/Monitor/FLMonitorViewController.swift
+++ b/Assets/Plugins/iOS/Floreo/Tabs/Monitor/FLMonitorViewController.swift
@@ -1031,7 +1031,8 @@ extension FLMonitorViewController: FLPlaylistCellDelegate {
     }
 
     func tryLoadGameplay(withID gameplayID: String) {
-        if let gameplay = (FLGameplayManager.sharedInstance.currentQueue.gamePlays.filter { $0.flid == gameplayID }).first {
+        let gameplays = FLGameplayManager.sharedInstance.currentQueue.gamePlays.filter { $0.flid == gameplayID }
+        if let gameplay = gameplays.first {
             tryLoadGameplay(gameplay)
         }
     }
-- 
```

**Comment by nicklockwood** [comment]
@alimoeeny fixed in 0.39.2. Thanks!

**Comment by alimoeeny** [comment]
Thanks a lot @nicklockwood  appreciate it.
